Classes/Settings/Settings.storyboard

```diff
@@ -125,7 +125,7 @@
                             </tableViewSection>
                             <tableViewSection headerTitle="🦅 Githawk" id="3QH-oa-l2C">
                                 <cells>
-                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" selectionStyle="default" accessoryType="disclosureIndicator" indentationWidth="10" textLabel="kdx-s1-NXl" style="IBUITableViewCellStyleDefault" id="WwO-m1-s3q" customClass="StyledTableCell" customModule="Freetime" customModuleProvider="target">
+                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" selectionStyle="default" accessoryType="disclosureIndicator" indentationWidth="10" textLabel="kdx-s1-NXl" style="IBUITableViewCellStyleDefault" id="WwO-m1-s3q" customClass="SpinnerTableCell" customModule="Freetime" customModuleProvider="target">
```

Comment:
Title here hasn't been changed, did we want to do that?

Reply:
I subclassed StyledTableCell to allow an activity indicator.

Reply:
I was more referring to the label inside of the cell, but can't add a review to a line outside of the change context 😉

Reply:
Oh haha! Ya ill fix that 😄Sorry